src/main/java/de/bonndan/nivio/input/Indexer.java

```diff
@@ -53,6 +53,7 @@ public ProcessLog reIndex(final LandscapeDescription input) {
             Map<ItemDescription, List<String>> templatesAndTargets = new HashMap<>();
             new SourceReferencesResolver(formatFactory, logger).resolve(input, templatesAndTargets);
             new TemplateResolver().processTemplates(input, templatesAndTargets);
+            new InstantItemResolver(logger).processTargets(input);
```

Comment:
@basseur @Matthimatiker @sl0815 

Any suitable patterns here?

Reply:
What do you mean? 😬

Reply:
new X, new Y, new Z ... and probably the group creation below might be handled by a class, too.